tests/torch/test_reduce_sum.py

```diff
@@ -59,18 +64,19 @@ def test_sum(
     check_circuit_has_no_tlu,
     check_circuit_precision,
     check_r2_score,
+    is_weekly_option,
 ):
     """Tests ReduceSum ONNX operator on a torch model."""
 
+    if has_tlu and not is_weekly_option:
```

Comment:
I've realized that reduce_sum tests were taking a long time for months because of compilation on 16-bits circuits. Maybe there is something to improve in the test but I found it simpler to keep these tests and put them in the weekly CI instead